src/hw25/Main.java

```diff
@@ -0,0 +1,44 @@
+package hw25;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        try {
```

Comment:
remove try catch from main. You are responsible to handle exceptions, not the client.